src/wcmCommon.c

```diff
@@ -890,7 +891,7 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 	if (!ds->proximity)
 		priv->flags &= ~SCROLLMODE_FLAG;
 
-	if (IsStylus(priv) && !IsArtPen(ds))
+	if (IsStylus(priv) && IsAirbrush(ds))
```

Comment:
Both ArtPen and Airbrush should be checked since they both need the 6th valuator: Artpen for rotation and Airbrush for finger wheel.

Reply:
Hi Ping, thank you for the feedback. 
The block following this one contains the following code: 

        } else if (IsStylus(priv) && IsArtPen(ds))
        {
                wcmAxisSet(&axes, WACOM_AXIS_WHEEL, ds->abswheel);
        }

That is, the code checking for the ArtPen already exists. The problem with the existing code was that it tried to use "!IsArtPen()" to catch the  "IsAirbrush()" condition and it ended up catching all pens other than the art pen. Perhaps I misunderstood your comment though.

Reply:
Looks correct, thanks, but I think this would be easier to read as:
```c
if (IsStylus(priv))
{
    if (IsArtPen(ds))
         ...
    else if (IsAirbrush(ds))
         ...
}
```
That way the current bug would've been more obvious too.